makepdf

```diff
@@ -117,13 +117,12 @@ while [ "${*}" != "" ] ; do
         if [[ ! -s "${TMP_PNG_DIR}/$(basename "${TIF%.*}").png" ]] ; then
             ffmpeg -i "${TIF}" -pix_fmt gray -y "${TMP_PNG_DIR}/$(basename "${TIF%.*}").png"
         fi
-        convert "${TMP_PNG_DIR}/$(basename "${TIF%.*}").png" -level 10%,75%,1 -blur 2 +dither -resize 2550x3300 -monochrome -flatten +repage -define png:bit-depth=8 -define png:color-type=0 "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}").png"
-        tesseract "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}").png" "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}")" -l eng -psm 4 pdf "${SCRIPTDIR}/tesseract.conf"
-        tesseract "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}").png" "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}")" -l eng -psm 4 "${SCRIPTDIR}/tesseract.conf"
-    done
+        convert "${TMP_PNG_DIR}/$(basename "${TIF%.*}").png" -blur 2 +dither -resize 1545x2000 -flatten +repage -define png:bit-depth=8 -define png:color-type=0 "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}").png"
+        tesseract "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}").png" "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}")" -l eng --psm 4 pdf "${SCRIPTDIR}/tesseract.conf"
+        tesseract "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}").png" "${TMP_CLEAN_DIR}/$(basename "${TIF%.*}")" -l eng --psm 4 "${SCRIPTDIR}/tesseract.conf"
     pdfjoin --fitpaper 'false' --rotateoversize 'false' --paper letter $(find "${TMP_CLEAN_DIR}" -name "*.pdf" | sort | xargs) --outfile "${OUTPUTDIR}/${MEDIAID}.pdf"
     cat $(find "${TMP_CLEAN_DIR}" -name "*.txt" | sort | xargs) > "${OUTPUTDIRTEXT}/${MEDIAID}.txt"
-
+done
```

Comment:
not sure why the end of the loop was moved, shouldn't this done be before before pdfjoin as it was before?

Reply:
it seemed to cause problems on the 22nd... it was specifically identified as a problem in the output after running the service... but adding it back in today it ran fine

Reply:
we struggled a lot with makepdf today and at the end of the day my local copy is (I believe) back to being identical to the live version.   I did figure out how to produce color pdfs and have samples of the color, greyscale and monochome pdfs with accompanying ocr text files to analyze - I'll email them to you